packages/api/src/routes/medical/internal-facility.ts

```diff
@@ -160,6 +183,9 @@ async function getCqFacilityName(oid: string) {
   return existingFacilityName;
 }
 
+function buildCwOboOrgName(vendorName: string, orgName: string, oboOid: string) {
+  return `${vendorName} - ${orgName} -OBO- ${oboOid}`;
+}
```

Comment:
Could just pass a param of HIE to update the - or #. Since they are pretty much the same but NBD

Reply:
They may have different rules in the future, and there has been some discussion around this - since they're distinct HIEs, I'll leave separate